docs/sdk/src/polkadot_sdk/polkadot.rs

```diff
@@ -83,5 +89,14 @@
 //!   Agile periodic-sale-based model for assigning Coretime on the Polkadot Ubiquitous Computer.
 //! - RFC#5: [Coretime-interface](https://github.com/polkadot-fellows/RFCs/blob/main/text/0005-coretime-interface.md):
 //!   Interface for manipulating the usage of cores on the Polkadot Ubiquitous Computer.
-// TODO: add more context and explanations about Polkadot as the Ubiquitous Computer and related
-// tech. https://github.com/paritytech/polkadot-sdk-docs/issues/66
+//!
+//! ### Polkadot as the Ubiquitous Computer
+//!
+//! - **Application Space**: for different parachains with their applications
+//! - **Accord Space**: for opt-in treaties across chains. Treaty's members cannot change/undermine
```

Comment:
```suggestion
//! - **Accord Space**: for opt-in treaties across chains. Treaty's members cannot change/undermine.
```

(nits, but would be nice to add a period at the end of all items below)